eacl/target.go

```diff
@@ -54,10 +75,41 @@ func (t *Target) BinaryKeys() [][]byte {
 //
 // Each element of the keys parameter is a slice of bytes is a serialized compressed public key.
 // See [elliptic.MarshalCompressed].
+// Deprecated: use [Target.SetAccounts] instead.
 func (t *Target) SetBinaryKeys(keys [][]byte) {
 	t.keys = keys
 }
 
+// Accounts returns list of accounts to identify target subject in a binary format.
+//
+// Each element of the resulting slice is a [user.IDSize] byte account.
+// Use `user := user.ID(slice)` to decode it into a type-specific structure.
+//
+// The value returned shares memory with the structure itself, so changing it can lead to data corruption.
```

Comment:
not critical, but it does not share memory, copying would be an overhead

Reply:
Removed